memq/src/main/java/com/pinterest/memq/core/rpc/BrokerTrafficShapingHandler.java

```diff
@@ -4,11 +4,14 @@
 import io.netty.handler.traffic.GlobalTrafficShapingHandler;
 
 import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
 import java.util.logging.Logger;
 
 public class BrokerTrafficShapingHandler extends GlobalTrafficShapingHandler {
 
+    public static final String READ_LIMIT_METRIC_NAME = "broker.traffic.read.limit";
     private static final Logger logger = Logger.getLogger(BrokerTrafficShapingHandler.class.getName());
+    private static int metricsReportingIntervalSec = 60; // default 1 minute
```

Comment:
This doesn't seem to be configurable. Correct?
If so, can we make it configurable?

Reply:
Configuration added.